.github/workflows/collector.yml

```diff
@@ -0,0 +1,24 @@
+name: cobrowse-evidence-flow
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  gh-pages:
+    name: Github Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-user@v1
```

Comment:
Is this action (fregante/setup-git-user) really needed? It doesn't seem to do much

Reply:
I'll try running it again without it.